modules/token-holders/pages/seed-donations.adoc

```diff
@@ -452,7 +452,7 @@ Call the governance canister to dissolve your neuron.
 [source,bash]
 ----
 CANISTER=rrkah-fqaaa-aaaaa-aaaaq-cai
-RESULT="$(dfx canister --network=https://ic0.app --no-wallet call $CANISTER manage_neuron "record { id = opt record { id = $NEURON_ID:nat64 }; command = opt variant { Configure = record { operation = opt variant { StartDissolving = record {} } } } }")"
+RESULT="$(dfx canister --network=https://ic0.app --no-wallet call $CANISTER manage_neuron "(record { id = opt record { id = $NEURON_ID:nat64 }; command = opt variant { Configure = record { operation = opt variant { StartDissolving = record {} } } } })" --output=raw)"
```

Comment:
Does this work or does the second double-quote terminate the first?

Reply:
yes

Reply:
FOOBAR="$(echo "test")"